docs/octopus_deploy_smtp_config.md

```diff
@@ -0,0 +1,37 @@
+# octopus_deploy_smtp_config
+
+Tests the server-wide SMTP configuration. This can be found in the UI under Configuration -> SMTP
```

Comment:
```suggestion
Tests the server-wide SMTP configuration. This can be found in the Octopus Portal under Configuration -> SMTP
```

_what if this moves in some UI reshuffle_?

Reply:
Do we need to say its a serverspec module?

Reply:
Probably not. There's a change to the root readme inbound later, and this file is linked from there. The whole shebang gets docs.